src/templates/home-page.js

```diff
@@ -182,7 +199,7 @@ export const query = graphql`
         }
       }
     }
-    allMarkdownRemark(
+    featuredNews: allMarkdownRemark(
```

Comment:
Seems like we're missing another labeled query for `latestResources`?

Reply:
🤔 something went awry here heh.

Reply:
Yeah, not sure how the `latestResources` query disappeared. I must have done something weird in my editor before committing. Also guarantee this is what's actually breaking the build.